nucypher/characters/lawful.py

```diff
@@ -544,7 +545,10 @@ def __bytes__(self):
         certificate = self.rest_server_certificate()
         cert_vbytes = VariableLengthBytestring(certificate.public_bytes(Encoding.PEM))
 
-        as_bytes = bytes().join((bytes(self._interface_signature),
+        timestamp = maya.now().epoch.to_bytes(4, 'big')
```

Comment:
Cool!  Looks like we're using epoch as to version ursula *each* time she is serialized?

Reply:
Yes, that's what @jMyles said to do. I think this was more of a thing when we hadn't removed the DHT, though.

Reply:
Yeah - its good. :+1:

Reply:
I gotta think more about this.